src/Write/Event/FeatureWasCreated.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pheature\Core\Toggle\Write\Event;
+
+final class FeatureWasCreated
+{
+
+    private string $eventType;
+
+    private string $featureId;
+
+    public function __construct(string $featureId)
```

Comment:
We could pass the `FeatureId` VO instead of the primitive to ensure that it's a right `FeatureId`. Then, inside the constructor we can assign the string value to the property.

Reply:
Is better to work with generics at this level to easy serialization of the object, we create the value object in the getter as it is immutable